src/applications/personalization/components/NameTag.jsx

```diff
@@ -175,9 +163,7 @@ const NameTag = ({
     ),
   };
 
-  const ariaLabel = showUpdatedNameTag
-    ? 'My status and disability rating'
-    : 'My status';
+  const ariaLabel = 'My information';
```

Comment:
nit: we could just get rid of this const now since it's not longer really helping readability down in the `section` tag.